docs/README.md

```diff
@@ -23,4 +23,4 @@ guidelines on how to get involved.
 
 ## License
 
-@poap-xyz/drops is released under the [MIT License](https://opensource.org/licenses/MIT).
+@poap-xyz/poap-sdk is released under the [MIT License](https://opensource.org/licenses/MIT).
```

Comment:
The license for the docs doesn't make sense, we can put Creative Commons one for the documentation.